src/__tests__/convert-spec.js

```diff
@@ -0,0 +1,62 @@
+import {convertContentTo, convertContentFrom} from '../lib/convert';
+import DraftJS, {ContentState} from 'draft-js';
+
+describe('convertContentTo', () => {
+
+	let draftJsContent;
+	beforeEach(() => {
+		draftJsContent = DraftJS.ContentState.createFromText('<div>some <a href="#">content</a></div>');
+	});
+
+	it('converts DraftJS content to json', () => {
+		expect(draftJsContent).toBeInstanceOf(ContentState);
+		let content = convertContentTo(draftJsContent, 'json');
+		expect(content).toBeInstanceOf(Object);
```

Comment:
Instead of comparing to Object instance, it's better to find out if the JSON result has a "blockMap" property..... If I remember correctly, After conversion you could have a JSON result that has a "blocks" property that will break the entire app (We created the entire conversion process in the past for that reason if you remember)

Reply:
I thought about doing it this way. That is probably better since Object is so vague.